### ISSUE TITLE
Convert visualization script to KnitR

### ISSUE THREAD

**Issue by NicMcPhee** [NicMcPhee/push-simplification-for-generalization#1]
It would be nice if `simplification_viz.R` was a KnitR file, as that would make it easier to runs parts of it separately, share on-line, etc.

Not a high priority item, but it would be nice to do.

**Comment by erp12** [comment]
Sure thing. That is the same thing as an R-markdown, right?

Just an FYI, I originally was working in an r-markdown document but asking r studio to draw the charts below each code block in the document did not work well with the larger images. We are going to scale down the images anyway, right?

**Comment by NicMcPhee** [comment]
Yup, KnitR is the same as R-markdown (and R-Markdown is probably the "better" name, actually).

For the paper we'll need PDFs of all the graphs, and we'll need to set their sizes to at least roughly match the desired size in the paper so the text scales appropriately. (If you make a really big plot, and then scale it down a lot in LaTeX, the text all becomes super tiny.)

**Comment by erp12** [comment]
Done with 7248f24906a5bdc45c437c9f07e6d1eae462c202.

@NicMcPhee Did you have a specific size to force the plots to be? Then I can adjust the font sizes, etc.
